### DISCUSSION TITLE
Pitch issue with flap

### DISCUSSION THREAD

**Discussion by kcire63** [flybywiresim/aircraft#2410] [Help]
Dear all,

When I arrived for landing, I reduce the speed to about 220Kts then I put flaps to 1. Then I have a pitch to nose down with pitch trim to full nose down, and a vario to 7000ft/mn to up....
A have to disconect AP I try to control it but it's not easy.
Thanks for you help.
I'm using the last version

**Comment by ghost** [comment]
Make sure that the game flight model is set to modern, and not legacy.

**Reply by kcire63** [reply]
Thank you for your answer but how i know that I am in Legacy or Modern mode?

**Reply by MapleNAV-TCASFAULT** [reply]
Its in the flight model page of general in options

**Reply by kcire63** [reply]
Thanks a lot, it works.

**Comment by Lucky38i** [comment]
Make sure you don't have icing.
